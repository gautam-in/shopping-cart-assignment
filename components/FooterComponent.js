import { FooterStyle } from "../styles/FooterStyle";

const FooterComponent = () => {
  return (
    <FooterStyle>
      <div>
        Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
      </div>
    </FooterStyle>
  );
};
export default FooterComponent;
