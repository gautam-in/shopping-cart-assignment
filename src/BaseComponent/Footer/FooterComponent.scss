/******************* footer *****************************/
@import '../../Styles/color-variable.scss';
@import '../../Styles/variable.scss';

.footer {
    width: 100%;
    min-height: $MEDIUM_HEIGHT;
    background-color: $LIGHT_GREY;
    font-size: $MEDIUM_FONT_SIZE;
    padding: $SMALL_PADDING;
    padding-left: $X_LARGE_PADDING;
}