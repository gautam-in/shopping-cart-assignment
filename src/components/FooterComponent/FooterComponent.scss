.footer-element {
    height: 54px;
    .footer_container {
      text-align: start;
      padding: 15px 125px;
      //padding: 15px;
      background-color: rgb(216, 216, 216);
      // position: fixed;
      bottom: 0px;
      width: 100%;
    }
  }