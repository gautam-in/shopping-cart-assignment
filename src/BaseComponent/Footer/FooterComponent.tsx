import "./FooterComponent.scss";
import { CONSTANT } from "../../Utility/Constant";

export const FooterComponent: React.FC = () => {
  const BASE_CLASSNAME: string = "footer";
  return (
    <footer className={BASE_CLASSNAME}>{CONSTANT.FOOTER_CONTENT_TEXT}</footer>
  );
};
