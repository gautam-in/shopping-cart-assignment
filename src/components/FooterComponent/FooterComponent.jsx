import "./FooterComponent.scss";
const FooterComponent = () => {
  return (
    <div className="footer-element">
      <footer className="footer_container m-text">
        Copyright Â© 2011-2022 Sabka Bazaar Grocery Supplies Pvt Ltd
      </footer>
    </div>
  );
};

export default FooterComponent;
