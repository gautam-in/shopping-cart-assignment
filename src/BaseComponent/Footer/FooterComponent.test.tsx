import { ShallowWrapper, shallow } from "enzyme";
import { FooterComponent } from "./FooterComponent";

describe("FooterComponent", () => {
    const tree: ShallowWrapper = shallow(
        <FooterComponent/>
    );

    test("FooterComponent Snapshot render", () => {
        expect(tree).toMatchSnapshot();
    });

    test("Check FooterComponent has classname", () => {
        expect(tree.hasClass("footer")).toEqual(true)
    });
});
