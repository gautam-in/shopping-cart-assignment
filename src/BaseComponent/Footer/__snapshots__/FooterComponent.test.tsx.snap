// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent FooterComponent Snapshot render 1`] = `ShallowWrapper {}`;
